library/ssl_tls13_server.c

```diff
@@ -1600,10 +1602,8 @@ static int ssl_tls13_parse_client_hello(mbedtls_ssl_context *ssl,
                 ret = mbedtls_ssl_parse_sig_alg_ext(
                     ssl, p, extension_data_end);
                 if (ret != 0) {
-                    MBEDTLS_SSL_DEBUG_MSG(1,
-                                          (
-                                              "ssl_parse_supported_signature_algorithms_server_ext ( %d )",
-                                              ret));
+                    MBEDTLS_SSL_DEBUG_MSG(
+                        1, ("mbedtls_ssl_parse_sig_alg_ext ( %d )", ret));
```

Comment:
```suggestion
                     MBEDTLS_SSL_DEBUG_RET(1, "mbedtls_ssl_parse_sig_alg_ext ", ret);
```

Reply:
Actually this will exceed 80 chars, even reach to 93 chars, which is what we will fix here.

Reply:
It should be replaced with `DEBUG_RET` .  If it exceed the limitation , format it.